openworm/org.geppetto.frontend.jupyter
# Feature/extensible routes

# What's new
With the feature extensible routes we are able to add new routes to the Jupyter server without the need to change the Jupyter extension itself.
Moreover, the path are now handled in a dynamic fashion. At the first call to */geppetto, the extension recognizes the base path and creates the relative routes, if needed.

# How to test
Installation has not changed from previous version.
The new code should not break NetPyNE and HNN.
A full setup with new routes has been implemented on NWB explorer, on branch feature/jupyter-backend

See README.md for more specifications on how to create controllers.
